prow/config.yaml

```diff
@@ -70,7 +70,7 @@ deck:
               - DATA RACE
               - dirty working directory
               - ^E\d{4} \d\d:\d\d:\d\d\.\d+
-              - "(Error|ERROR|error)s?:"
+              - "(Error|ERROR|error)s?(:|\s)"
```

Comment:
```suggestion
              - '(Error|ERROR|error)s?(:|\s)'
```

Reply:
Hi @Halamix2 , thanks! I removed the quote completely